plugins/nodes/src/js/pages/nodes/nodes-grid/NodesGridContainer.js

```diff
@@ -11,6 +11,7 @@ import QueryParamsMixin from '../../../../../../../src/js/mixins/QueryParamsMixi
 
 const MAX_SERVICES_TO_SHOW = 32;
 const METHODS_TO_BIND=['handleShowServices'];
+const OTHER_SERVICES_COLOR = 32;
```

Comment:
this is correct. All the colors are defined in `src/styles/components/nodes-grid-view/styles.less`